.github/workflows/build-and-test.yaml

```diff
@@ -198,8 +199,8 @@ jobs:
       timeout-minutes: 10
       working-directory: build
       run: |
-        ./tests/test-erlang
-        valgrind ./tests/test-erlang
+        ./tests/test-erlang ${{ matrix.test_erlang_skips }}
```

Comment:
I thought the test only failed with Valgrind + 32 bits.

Reply:
It also fails on Ubuntu 18 with gcc 4.8 on i386.  Since Ununtu 18 is going away, I don't think we need to worry about it.